common/action-manager.js

```diff
@@ -46,7 +46,9 @@ const am = new ActionManager([
   "NOTIFY_TOGGLE_RECOMMENDATIONS",
   "RECEIVE_RECOMMENDATION_TOGGLE",
```

Comment:
The code for this action & tests have been removed. Are we still using this somewhere?